pkg/aks/create.go

```diff
@@ -110,21 +110,17 @@ func CreateOrUpdateCluster(ctx context.Context, cred *Credentials, clusterClient
 		}
 	}
 
-	var addonProfiles map[string]*containerservice.ManagedClusterAddonProfile
+	addonProfiles := map[string]*containerservice.ManagedClusterAddonProfile{}
```

Comment:
`make` is preferrable in this caseper our standards (which are basically https://github.com/uber-go/guide/blob/master/style.md#initializing-maps)